3.10/security-security-options.md

```diff
@@ -348,6 +348,16 @@ extra options are available for locking down JavaScript access to server functio
 The following options are available for controlling the installation of Foxx applications
 in an ArangoDB server:
 
+- `--foxx.enable`:
+  If set to `false`, this option disables access to any user-defined Foxx apps.
+  Accessing the URL of any (existing or potentially existing) Foxx app will produce an
+  HTTP 403 Forbidden error with this setting.
```

Comment:
Not sure how we should format error codes in general. In this specific case, it could also be inline code for the part you would actually see in the HTTP request.
```suggestion
  Accessing the URL of any (existing or potentially existing) Foxx app produces an
  HTTP **403 Forbidden** error with this setting.
```

Reply:
In the Google style guide, it is recommended to use inline codes for error codes/status codes.